llama_hub/preprocess/base.py

```diff
@@ -0,0 +1,145 @@
+"""Preprocess Reader."""
+import os
+import hashlib
+from typing import List
+from llama_index.readers.base import BaseReader
+from llama_index.readers.schema.base import Document
+from llama_index.schema import TextNode, NodeRelationship, RelatedNodeInfo
+
+from pypreprocess import Preprocess
```

Comment:
typically we move this import under the __init__ method of the Reader. See for example: https://github.com/run-llama/llama-hub/blob/68fed88d8e36223c6d8fa50398b347add84cd66f/llama_hub/confluence/base.py#L41

Reply:
Can you kindly do the same?

Reply:
its okay @preprocess-co  — I took care of it. All checks pass now. Thanks for the contribution! merging now :)

Reply:
@nerdai Thank you, sorry i didn't notice it on time to handle it :)